packages/venia-ui/lib/components/MiniCart/ProductList/productList.js

```diff
@@ -0,0 +1,63 @@
+import React, { useMemo } from 'react';
+import { string, func, arrayOf, shape, bool, number } from 'prop-types';
+
+import LoadingIndicator from '../../LoadingIndicator';
+import Item from './item';
+import { mergeClasses } from '../../../classify';
+
+import defaultClasses from './productList.css';
+
+const ProductList = props => {
+    const { items, loading, handleRemoveItem, classes: propClasses } = props;
+    const classes = mergeClasses(defaultClasses, propClasses);
+
+    const cartItems = useMemo(() => {
+        if (!loading && items) {
+            return items.map(item => (
+                <Item
+                    key={item.id}
+                    {...item}
+                    handleRemoveItem={handleRemoveItem}
+                />
+            ));
+        }
+    }, [items, loading, handleRemoveItem]);
+
+    if (loading) {
```

Comment:
Was the desire to display this loading indicator when we remove items? It feels a little excessive. Maybe run it by UX?

[![Image from Gyazo](https://i.gyazo.com/ac33311bf57e1ae48bc4102b4eb512d8.gif)](https://gyazo.com/ac33311bf57e1ae48bc4102b4eb512d8)

Reply:
This is definitely quite extra. @schensley had done work on loading indicators a while ago I think. Could we use a pattern from that here?

Reply:
I had hoped for the background loading indicator to have some use here, replacing some or all of the content in the minicart's header (count and/or subtotal) Is it not usable here?

Reply:
What do you guys think should be the alternative here? Should we handle it here or as part of a different ticket? I am not sure if the loading indicator is part of the original ticket, so if possible, it can be handled in a new one instead of expanding the scope of the current.

Reply:
@soumya-ashok @schensley I have removed the loading indicator while deleting an item in the cart. The new behavior is to remove the delete icon and change the opacity of the item to 0.5 to indicate that it is being deleted. Check it out and let me know if that is how we want to handle it.

Reply:
Looks pretty good to me.